classes/main.rb

```diff
@@ -33,15 +33,15 @@ def load_json_file(filename)
     JSON.parse(data) unless data.empty?
   end
 
-  def display_books
-    books = load_json_file('books.json')
-    if books
-      puts 'Books:'
-      books.each { |book| display_book(book) }
-    else
-      puts 'No books available.'
-    end
-  end
+  # def display_books
+  #   books = load_json_file('books.json')
```

Comment:
- [ ] [REQUIRED] Good job so far! Make sure that the role of main.rb is only to start the code and ask the user for the initial choice of options but you have a error on your `main.rb` because when run your code it shows to error, please fix the error 🙏🏼 

![image](https://github.com/Simpleshaikh1/OOP-SCHOOL-LIBRARY-RUBY/assets/63915024/bf1907c1-6322-46a8-84f1-1c2284a5d4dc)